src/wp-content/themes/twentytwelve/functions.php

```diff
@@ -241,17 +281,6 @@ function twentytwelve_block_editor_styles() {
  * @return array URLs to print for resource hints.
  */
 function twentytwelve_resource_hints( $urls, $relation_type ) {
-	if ( wp_style_is( 'twentytwelve-fonts', 'queue' ) && 'preconnect' === $relation_type ) {
-		if ( version_compare( $GLOBALS['wp_version'], '4.7-alpha', '>=' ) ) {
-			$urls[] = array(
-				'href' => 'https://fonts.gstatic.com',
-				'crossorigin',
-			);
-		} else {
-			$urls[] = 'https://fonts.gstatic.com';
-		}
-	}
-
 	return $urls;
 }
 add_filter( 'wp_resource_hints', 'twentytwelve_resource_hints', 10, 2 );
```

Comment:
The resource hint filter should not return an empty string for everyone. We could simply remove the `add_filter` line and add a `deprecated` comment in the documentation (under `since`). Then anyone who still has a custom Google URL could add this pre-defined filter in a child theme or plugin.

Reply:
I've updated the PR to remove the whole filter function. I'm unsure about the deprecation note, can you maybe take a look at it, @sabernhardt?

Reply:
My point was to [leave the `twentytwelve_resource_hints` function intact](https://github.com/sabernhardt/wordpress-develop/pull/12/files#diff-ee4bfe2db41f6d65222cb7cb8d2c59040db603c23afbd7321b6fd743a5c0c5c7R214). The version number in that example PR is outdated, but I only added one line to the documentation and removed the one line that calls the function.

Child themes can still use Google URLs, especially if their font selection is different, and some of those sites' owners may want to continue employing that filter.

Reply:
Should we just comment out the filter?

Reply:
Commenting out the line might be better. That would show what was removed (and could be restored) even without explaining it in the DocBlock.

Reply:
I've just adapted the changes from your PR to this PR. I hope I got it all.